package.json

```
@@ -37,6 +37,23 @@
     "LICENSE.txt"
   ],
   "main": "qunit/qunit.js",
+  "exports": {
+    ".": {
+      "node": {
+        "import": "./qunit/esm/qunit-wrapper-nodejs-module.js",
+        "default": "./qunit/qunit.js"
+      },
+      "module": {
+        "import": "./qunit/esm/qunit.module.js",
+        "default": "./qunit/qunit-wrapper-bundler-require.js"
+      },
+      "import": "./qunit/esm/qunit.module.js",
+      "default": "./qunit/qunit.js"
+    },
+    "./qunit/qunit.css": {
+      "default": "./qunit/qunit.css"
+    }
```

If you have a single export, you can just provide the path to the entry point directly:
```suggestion
    "./qunit/qunit.css": "./qunit/qunit.css"
```